### PR TITLE
Autofill input field of file upload name on Patients page

### LINKED ISSUE #3034: File name not autofilling when uploaded on Patients page
https://care.coronasafe.in/facility/c92cc4df-0fea-4d37-9c24-1c9f07bfac23/patient/36357bfe-c143-4944-bab3-7e4be0a73e89/consultation/79421615-b734-400d-b361-4ae5bd9539a0/files

https://www.loom.com/share/a65cb8643ba9456d90739d9c3731b8e1

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #3034 

https://www.loom.com/share/646f12507cf142a5acc33be3beb99444

### CODE REVIEW

File: src/Components/Patient/FileUpload.tsx [hunk 1]
```diff
@@ -391,8 +391,9 @@ export const FileUpload = (props: FileUploadProps) => {
     if (e.target.files == null) {
       throw new Error("Error finding e.target.files");
     }
-    const f = e.target.files[0];
-    const fileName = f.name;
+    setfile(e.target.files[0]);
```
Comment (rithviknishad) [inline src/Components/Patient/FileUpload.tsx]: image

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@Pranshu1902 remove the file extension from the name autofill

**Comment by Pranshu1902** [comment]
@nihal467 done

**Comment by nihal467** [comment]
test approved

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**